undertow/WFLY-9811_Pre-Compressed_Resources.adoc

```diff
@@ -0,0 +1,64 @@
+= WFLY-9811 Pre-compressed Resources
+:author:            Stuart Douglas
+:email:             sdouglas@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org
```

Comment:
It's probably like that in the template...

Reply:
the template in this repo doesn't use **issue-base-url variable**, but puts the whole link in the text.
The other option is to add **"/browse"** to the text instead of here, like **"{issue-base-url}/browse/WFLY-9811[WFLY-9811]"**.